lib/taskana-core/src/test/java/acceptance/task/complete/CompleteTaskAccTest.java

```diff
@@ -1,118 +1,159 @@
+    assertThat(completedTask.getCompleted())
+        .isEqualTo(completedTask.getModified())
+        .isNotNull()
+        .isNotEqualTo(completedTask.getCreated());
   }
 
   @WithAccessId(user = "admin")
   @WithAccessId(user = "taskadmin")
   @TestTemplate
-  void should_ForceCompleteTask_When_NoExplicitPermissionsButUserIsInAdministrativeRole()
+  void should_ForceCompleteClaimedTask_When_NoExplicitPermissionsButUserIsInAdministrativeRole()
       throws Exception {
-    resetDb(false);
+    Task claimedTask = createTaskClaimedByUser().buildAndStore(taskService);
+
+    Task completedTask = taskService.forceCompleteTask(claimedTask.getId());
 
-    assertThat(taskService.getTask("TKI:000000000000000000000000000000000000").getState())
-        .isEqualTo(TaskState.CLAIMED);
-    Task completedTask = taskService.forceCompleteTask("TKI:000000000000000000000000000000000000");
     assertThat(completedTask).isNotNull();
     assertThat(completedTask.getCompleted()).isNotNull();
     assertThat(completedTask.getState()).isEqualTo(TaskState.COMPLETED);
-    assertThat(completedTask.getModified()).isNotEqualTo(completedTask.getCreated());
+    assertThat(completedTask.getCompleted())
+        .isEqualTo(completedTask.getModified())
+        .isNotNull()
+        .isNotEqualTo(completedTask.getCreated());
   }
 
   @WithAccessId(user = "user-1-1")
   @Test
-  void testCompleteTaskTwice() throws Exception {
-    Task completedTask = taskService.completeTask("TKI:000000000000000000000000000000000002");
-    Task completedTask2 = taskService.completeTask("TKI:000000000000000000000000000000000002");
+  void should_CompleteTaskTwice() throws Exception {
+    Task claimedTask = createTaskClaimedByUser().buildAndStore(taskService);
+
+    Task completedTask = taskService.completeTask(claimedTask.getId());
+    Task completedTask2 = taskService.completeTask(claimedTask.getId());
+
     assertThat(completedTask2).isEqualTo(completedTask);
   }
 
   @WithAccessId(user = "user-1-1")
   @Test
-  void testForceCompleteAlreadyClaimed() throws Exception {
-    Task newTask = taskService.newTask("USER-1-1", "DOMAIN_A");
-    newTask.setClassificationKey("T2100");
-    newTask.setOwner("other");
-    newTask.setPrimaryObjRef(
-        createObjectReference("COMPANY_A", "SYSTEM_A", "INSTANCE_A", "VNR", "1234567"));
-    TaskImpl newTaskImpl = (TaskImpl) newTask;
-    newTaskImpl.setState(TaskState.CLAIMED);
-    newTaskImpl.setClaimed(Instant.now());
+  void should_ForceCompleteClaimedTask_When_TaskCallerIsNotTheOwner() throws Exception {
+    Task task = createTaskClaimedByUser().owner("other").buildAndStore(taskService);
 
-    Task createdTask = taskService.createTask(newTaskImpl);
-    Task completedTask = taskService.forceCompleteTask(createdTask.getId());
+    Task completedTask = taskService.forceCompleteTask(task.getId());
 
     assertThat(completedTask.getState()).isEqualTo(TaskState.COMPLETED);
     assertThat(completedTask.getCompleted()).isNotNull();
```

Comment:
since the assert statements two lines below already contains `isNotNull` this is not necessary.